README.md

```diff
@@ -327,9 +327,20 @@ Supabase.instance.client.auth.signInWithProvider(
 
 ### Custom LocalStorage
 
-As default `supabase_flutter` uses [`hive`](https://pub.dev/packages/hive) plugin to persist user session. However you can use any other plugins by creating a `LocalStorage` impl.
+As default, `supabase_flutter` uses [`hive`](https://pub.dev/packages/hive) to persist user session. Encryption is disabled by default, since it's necessary a unique key we can not define. To set an `encryptionKey`, do the following:
```

Comment:
```suggestion
As default, `supabase_flutter` uses [`hive`](https://pub.dev/packages/hive) to persist user session. Encryption is disabled by default, since a unique encryption key is necessary, and we can not define it. To set an `encryptionKey`, do the following:
```